src/components/user_view/UserInfo.js

```diff
@@ -0,0 +1,29 @@
+import React, { Component } from 'react';
+
+export default class UserInfo extends Component {
```

Comment:
I'm probably just missing something, but are you actually using this class and its container anywhere? I'm just seeing where you are using the display and edit versions

Reply:
Negative! Will remove this. This was deleted in my local branch but somehow snuck it's way into this remote one.